core/src/test/java/apoc/trigger/TriggerNewProceduresTest.java

```diff
@@ -499,6 +505,35 @@ public void testDeleteRelationships() {
     //
     // new test cases
     //
+
+    @Test
+    public void testTriggerShow() throws Exception {
+        String name = "test-show1";
+        String name2 = "test-show2";
+        String query = "MATCH (c:TestShow) SET c.count = 1";
+        
+        testCall(sysDb, "CALL apoc.trigger.install('neo4j', $name, $query,{}) YIELD name",
+                map("query", query, "name", name),
+                r -> assertEquals(name, r.get("name")));
+        
+        testCall(sysDb, "CALL apoc.trigger.install('neo4j', $name, $query,{}) YIELD name",
+                map("query", query, "name", name2),
+                r -> assertEquals(name2, r.get("name")));
+        
+        // not updated
+        testResult(sysDb, "CALL apoc.trigger.show('neo4j') " +
+                        "YIELD name, query RETURN * ORDER BY name",
+                map("query", query, "name", name),
+                res -> {
+                    Map<String, Object> row = res.next();
+                    assertEquals(name, row.get("name"));
+                    assertEquals(query, row.get("query"));
+                    row = res.next();
+                    assertEquals(name2, row.get("name"));
+                    assertEquals(query, row.get("query"));
+                    assertFalse(res.hasNext());
+                });
+    }
```

Comment:
Could we also add a test where we install a procedure on a database "foo" and then assert that it isn't returned when showing triggers from database "neo4j"?